src/ucs/config/global_opts.c

```diff
@@ -67,7 +67,8 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "UCS logging level. Messages with a level higher or equal to the selected "
   "will be printed.\n"
   "Possible values are: fatal, error, warn, info, debug, trace, data, func, poll.",
-  ucs_offsetof(ucs_global_opts_t, log_level), UCS_CONFIG_TYPE_ENUM(ucs_log_level_names)},
+  ucs_offsetof(ucs_global_opts_t, log_component),
+  UCS_CONFIG_TYPE_COMP},
```

Comment:
why need to define it? Can't you just change the log level? You are not supposed to change the component name, right?

Reply:
UCS_CONFIG_TYPE_COMP defines clone method in order to ucs_config_parses_clone_opts works properly